plugins/flashrom/fu-plugin-flashrom.c

```diff
@@ -41,7 +40,6 @@ fu_plugin_init (FuPlugin *plugin)
 	fu_plugin_add_rule (plugin, FU_PLUGIN_RULE_METADATA_SOURCE, "linux_lockdown");
 	fu_plugin_add_rule (plugin, FU_PLUGIN_RULE_CONFLICTS, "coreboot"); /* obsoleted */
 	fu_plugin_add_flag (plugin, FWUPD_PLUGIN_FLAG_REQUIRE_HWID);
-	fu_plugin_add_device_gtype (plugin, FU_TYPE_FLASHROM_LSPCON_I2C_SPI_DEVICE);
 	fu_plugin_add_udev_subsystem (plugin, "i2c");
```

Comment:
This line should also go away... Following the opposite from fe073a304ddd4db10c6ca2de75a649cad65e1a9a

Reply:
Good catch; done.